.claude/commands/exportcommands.py

```
@@ -455,23 +462,21 @@ def _detect_version(self):
         except Exception as e:
             print(f"   ⚠️ GitHub repository version detection failed: {e}")
             pass
-        
+
         # For this specific export, we want to use v1.1.0 for today's changes
         # regardless of what's in the target repository
         version = "1.1.0"
         print(f"   📋 Using v1.1.0 for command count consistency fixes")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)